src/styles/index.scss

```diff
@@ -1,3 +1,124 @@
+$main-accent-color: #060b35;
+$secondary-color: #616070;
+$blue-accent-color: #00acdc;
+
 body {
   margin: 0;
+
+  color: $main-accent-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+}
+
+.product-card {
+  width: 165px;
+
+  padding: 32px 16px 16px;
+
+  background-color: #fff;
+  color: $main-accent-color;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image-container {
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+
+    margin-bottom: 30px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+
+    width: 166px;
+  }
+
+  &__code {
+    color: $secondary-color;
+
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+  }
+
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__star {
+    width: 20px;
+    height: 20px;
+  }
+
+  &__reviews-text {
+    font-size: 10px;
+
+    margin-left: 8px;
+  }
+
+  &__price-label {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 400;
+
+    color: $secondary-color;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    line-height: 18px;
+
+    font-weight: 700;
+
+    color: #000;
+  }
+
+  &__buy-button {
+    width: 100%;
+    height: 40px;
+
+    border-radius: 5px;
+    border: none;
+
+    background: var(--blue-accent-color, #00acdc);
```

Comment:
The use of `var(--blue-accent-color, #00acdc)` is incorrect because the variable `--blue-accent-color` is not defined in the SCSS file. Instead, you should use the SCSS variable `$blue-accent-color` directly: `background: $blue-accent-color;`.